queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name = '%mon';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-01-01';
+SELECT * FROM animals WHERE neutered = true AND escape_attempts < 3;
```

Comment:
- [ ] To satisfy the query results, I recommend you select only the animals' `name` and not their entire, `*` details.
- [ ] `SELECT name FROM animals ...`